templates/frontend/pages/indexJournal.tpl

```diff
@@ -56,22 +56,21 @@
 									<p>
 										{capture assign="announcementPageUrl"}{url router=$smarty.const.ROUTE_PAGE page="announcement" op="view" path=$announcement->getId()}{/capture}
 										<a href="{$announcementPageUrl}" class="btn btn-secondary">{translate key="common.more"}</a>
-
-										{* Carousel controls *}
-										<span class="float-right">
-											<a href="#announcementsCarouselControls" class="btn" role="button" data-slide="prev">
-												<span aria-hidden="true">←</span>
-												<span class="sr-only">{translate key="help.next"}</span>
-											</a>
-											<a href="#announcementsCarouselControls" class="btn" role="button" data-slide="next">
-												<span aria-hidden="true">→</span>
-												<span class="sr-only">{translate key="help.previous"}</span>
-											</a>
-										</span>
 									</p>
 								</article>
 						{/foreach}
 					</div>
+					{* Carousel controls *}
```

Comment:
Don't you think it makes sense to show the controls only if a number of announcements to show is more than 1?

Reply:
Fixed and pushed!